pandas/compat/_optional.py

```diff
@@ -72,6 +73,10 @@ def get_version(module: types.ModuleType) -> str:
         if module.__name__ == "brotli":
             # brotli doesn't contain attributes to confirm it's version
             return ""
+        if module.__name__ == "snappy":
+            # snappy doesn't contain attributes to confirm it's version
```

Comment:
`snappy.__version__` for me says "3.0.3"

Reply:
Are you confusing it with https://snappy.math.uic.edu/?

Reply:
Yes, good catch!